config/envs/default.py

```diff
@@ -40,6 +40,9 @@ class DefaultConfig:
     APPLICATION_STATUS_ENDPOINT = "/applications/{application_id}/status"
     APPLICATION_SEARCH_ENDPOINT = "/applications?{params}"
 
+    #Assesment store endpoints
+    ASSESSMENT_SCORE_JUST_ENDPOINT = "{host}/assessments/{assessment_id}/sub_criterias/{sub_criteria_id}/scores"
```

Comment:
- IMHO its better to have the whole word justification here.
- why not include ASSESSMENT_STORE_API_HOST here instead of formatting in code